src/index.es

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file index 为所有const对象添加Object.freeze
+ * @author zongyu(liuxuanzy@qq.com)
+ */
+
+import template from 'babel-template';
+
+let buildFactory = template(`Object.freeze($0)`);
+
+module.exports = function ({types: t}) {
```

Comment:
这个最好和其它社区里的babel插件保持一致，方便别人使用

> 在 2016年3月26日，下午5:39，Yang Huabei <notifications@github.com> 写道：
> 
> In src/index.es:
> 
> > @@ -0,0 +1,39 @@
> > +/**
> > + * @file index 为所有const对象添加Object.freeze
> > + * @author zongyu(liuxuanzy@qq.com)
> > + */
> > +
> > +import template from 'babel-template';
> > +
> > +let buildFactory = template(`Object.freeze($0)`);
> > +
> > +module.exports = function ({types: t}) {
> 可以考虑使用es6定义模块，反正要用babel转码
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>